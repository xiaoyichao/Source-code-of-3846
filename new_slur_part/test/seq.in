I'd like to listen to Kinokoteikoku
I'd like to listen to Justin Timberlake
I want the lights to be at level twenty two
I'd like to listen to Pharrell Williams
I want my cubicle lights to be on two.
I'd like to listen to Malaka
rainy day songs play now please
Can I please listen to the first track from selena gomez.
I'd like to listen to Russ
I'd like to listen to Carlos Cid
Turn the vestibule lights to two.
I'd like to listen to Jenny Berggren
I'd like to listen to Michael Patrick Kelly
I'd like to listen to Glass Animals
Change all of the garden lighting to twenty
Make the pantry five percent in brightness
I'd like to listen to Golden Features
I'd like to listen to Nina Sky
I'd like to listen to Niall Horan
I'd like to listen to Kajo
I'd like to listen to Fakkboiz
I'd like to listen to Rocío Garcel
I'd like to listen to Rombai
I'd like to listen to Yoga Lin
I'd like to listen to Gusi
I want the brightness to be set at twenty.
I'd like to listen to Alejandro Vazquez
Adjust the cella lights to twenty two
volume lower to 11%
I'd like to listen to Yasha
I'd like to listen to Tia Ray
Switch the lights to dark in the pantry
Please turn the living room lights to twelve.
Decrease the light intensity to zero
I'd like to listen to Pamela
I'd like to listen to Mariachi México 11 De Pepe López
I'd like to listen to Lana Del Rey
Turn the brightness in the flat to two
I'd like to listen to Macky
I'd like to listen to Florida Georgia Line
Lower volume by three
I'd like to listen to James Morrison
volume to 11
I'd like to listen to Kinderen Voor Kinderen
I'd like to listen to Boygar
I'd like to listen to Zara Larsson
I'd like to listen to Chris Brown
Turn the studio lights to thirty nine
queue the weekend
I'd like to listen to Phoebe Ryan
set brightness of house to fifty
Adjust the glow of the bulb to level two
Increase the brightness to seventy-three.
I'd like to listen to Ronald Cheng
Change all the lights to fifty.
I'd like to listen to Meiko
I'd like the brightness set to twenty
I'd like to listen to G Herbo
I'd like to listen to Sokrat ST
I'd like to listen to Daynik
I'd like to listen to Susan Boyle
I'd like to listen to Heinis
Turn up the speakers by six
I'd like to listen to Majid Jordan
increase volume to fifty
I'd like to listen to Sofia Reyes
I'd like to listen to Lookas
Turn the waiting room brightness to two
I'd like to listen to Salif Keita
I'd like to listen to Ale Zurita
I'd like to listen to Mauro Tejeiro
I'd like to listen to Cholo Valderrama
I want the vestibule to shine at twelve percent
I'd like to listen to Enginarter
I'd like to listen to Nino
I'd like to listen to Möwe
I'd like to listen to Kai
Adjust the laundry room lighting to level thirty nine
I'd like to listen to Die Lochis
I'd like to listen to Siti Nordiana
Turn up the volume six more notches
it's too quiet, make it two levels more
I'd like to listen to Ronnie Flex
set the volume a bit louder, to eighty
I'd like to listen to Madeline Juno
I'd like to listen to Camila Queiroz
Help me set the entrance light to two
I would like to turn the volume down by three.
I'd like to listen to Paulo Londra
I'd like to listen to Vanrip
Make the volume higher by six.
I'd like to listen to Shabu
I'd like to listen to Polina
play merriweather post pavillion
I'd like to listen to Burak Yeter
I'd like to listen to Davay
Set the brightness level of light to twenty
I'd like to listen to Sebastian
I'd like to listen to Jess Glynne
I'd like to listen to Genuino
I'd like to listen to DJ Arow
Search for kris wu songs.
I'd like to listen to Danny Ocean
I'd like to listen to Jiggy Dje
I'd like to listen to KOI
I'd like to listen to Skítamórall
Adjust the toilet lighting to the two setting
raise the volume to eighteen percent
I'd like to listen to Zac Efron
I'd like to listen to Tito Fernández
Play my reba mcentire songs.
I'd like to listen to Arcangel
I'd like to listen to Kafu Banton
I'd like to listen to Jonas Blue
Volume up by four
I'd like to listen to Max Jenmana
I'd like to listen to Serebro
I'd like to listen to Peabo Bryson
Please play the song the weekend.
I'd like to listen to DJ Smallz 111
I'd like to listen to Vanessa Williams
Switch off all exterior lighting.
Switch to red lights
I'd like to listen to Ersan Er
I'd like to listen to Cem Belevi
I'd like to listen to Rino Gaetano
I'd like to listen to Felix Jaehn
Lower the volume by two
I'd like to listen to Calogero
I'd like to listen to Mørland
I'd like to listen to NF
I'd like to listen to Nagita
Find Music by julia Michaels?
set brightness to five
I'd like to listen to Marama
Please play simon and garfunkel songs.
I'd like to listen to Gustas Mio
I'd like to listen to Beau Monga
i'd like to listen to the discovery weekly playlist in shuffle mode
play kendrick lamar's songs
I'd like to listen to K. Michelle
I'd like to listen to Barbora Poláková
I want the cublicle to be at thirty nine brightness.
I'd like the vestibule lighting to be at thirty nine
I'd like to listen to R. City
I'd like to listen to Shehyee
I'd like to listen to Josylvio
I'd like to listen to MC Fioti
increase loudness by seven
please set the volume down to 11
What do I need to do to get the dining room to fifty brightness?
I'd like to listen to Salim
I'd like to listen to Yo Gotti
I'd like to listen to Blackbear
I'd like to listen to Luqman Faiz
change volume level to 11%
I'd like to listen to Kygo
volume up to 11
I'd like to listen to Snakehips
I'd like to listen to Young the Giant
I'd like to listen to Parson James
I'd like to listen to Nakor
I'd like to listen to Justyna Święs
Switch the lights green.
I'd like to listen to The Greatest Showman Ensemble
Make the lights in the cellar at the two brightness.
I'd like to listen to The Supermen Lovers
I'd like to listen to Leipe Dennis
Decrease the light intensity in the entrance to zero
I'd like to listen to Lionel Richie
I'd like to listen to Tivi Gunz
Can you set the bathroom brightness level to thirty nine?
I'd like to listen to Dalida
lower tv volume to 11
can you increase volume by one
I'd like to listen to Lhast
Set the brightness on the light bulbs to fifty.
I want the kids bedroom to be set to level fifty.
I'd like to listen to Tanya Stephens
Play the songs of leonard cohen on repeat.
I would like the garage lights on fifty.
volume down to 11%
I'd like to listen to Jugglerz
I'd like to listen to DJ Myst
play alan jackson
