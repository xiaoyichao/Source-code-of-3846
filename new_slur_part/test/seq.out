O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O O O O B-brightness I-brightness
O O O O O B-artist_name I-artist_name
O O O B-room O O O O B-brightness
O O O O O B-artist_name
B-playlist_name I-playlist_name O O O O
O O O O O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O B-room O O B-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-room O O B-brightness
O O B-room B-brightness O O O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O O O O B-brightness
O O O O O B-artist_name I-artist_name
O O B-room O O B-brightness I-brightness
O O O B-volume_level_percent
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O O O B-room
O O O B-room I-room O O B-brightness
O O O O O O
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name I-artist_name I-artist_name I-artist_name I-artist_name
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-room O B-brightness
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name I-artist_name
O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O B-room O O B-brightness I-brightness
O B-song_name I-song_name
O O O O O B-artist_name I-artist_name
O O O B-room O B-brightness
O O O O O O O O B-brightness
O O O O B-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-brightness
O O O O O B-artist_name
O O O O O O B-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O B-room I-room O O B-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O B-room O O O B-brightness O
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O B-room I-room O O O B-brightness I-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-volume_level_absolute O O
O O O O O B-volume_level_absolute O O
O O O O O B-artist_name I-artist_name
O O O O O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-room O O B-brightness
O O O O O O O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-volume_level_absolute
O O O O O B-artist_name
O O O O O B-artist_name
O B-album_name I-album_name I-album_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O O O B-brightness
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O B-artist_name I-artist_name O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O B-room O O O B-brightness O
O O O O B-volume_level_percent I-volume_level_percent
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O B-artist_name I-artist_name O
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O B-volume_level_absolute
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-song_name I-song_name
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O
O O B-color O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-volume_level_absolute
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O O B-artist_name I-artist_name
O O O B-brightness
O O O O O B-artist_name
O O B-artist_name I-artist_name I-artist_name O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O O B-playlist_name I-playlist_name O O B-playlist_mode O
O B-artist_name I-artist_name O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O B-room O O O B-brightness I-brightness O
O O O B-room O O O O B-brightness I-brightness
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O B-volume_level_absolute
O O O O O O B-volume_level_absolute
O O O O O O O O O B-room I-room O B-brightness O
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-volume_level_percent
O O O O O B-artist_name
O O O B-volume_level_absolute
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O O O B-color
O O O O O B-artist_name I-artist_name I-artist_name I-artist_name
O O O O O B-room O O B-brightness O
O O O O O B-artist_name I-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O O O B-room O O
O O O O O B-artist_name I-artist_name
O O O O O B-artist_name I-artist_name
O O O O B-room O O O B-brightness I-brightness
O O O O O B-artist_name
O O O O B-volume_level_absolute
O O O O O B-volume_level_absolute
O O O O O B-artist_name
O O O O O O O O B-brightness
O O O B-room I-room O O O O O B-brightness
O O O O O B-artist_name I-artist_name
O B-album_name I-album_name I-album_name I-album_name I-album_name O B-playlist_mode
O O O O B-room O O B-brightness
O O O B-volume_level_percent
O O O O O B-artist_name
O O O O O B-artist_name I-artist_name
O B-artist_name I-artist_name
